### PR TITLE
fix: nginx con't find fateflow when fateflow restart

### PR DESCRIPTION
**Pull request by owlet42**

### CODE REVIEW

File: helm-charts/FATE/templates/python-spark.yaml [hunk 1]
```diff
@@ -301,7 +305,7 @@ spec:
               subPath: logs
             - mountPath: /data/projects/fate/conf1/
               name: python-confs
-            - mountPath: /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
+            - mountPath: /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults-bak.conf
```
Comment (LaynePeng) [inline helm-charts/FATE/templates/python-spark.yaml:308]: why "-bak"?

**Review by LaynePeng — CHANGES_REQUESTED** [review]

File: helm-charts/FATE/templates/python-spark.yaml (hunk 1 above)
Comment (jiahaoc1993) [reply helm-charts/FATE/templates/python-spark.yaml:308]: Same question from my side

Comment (owlet42) [reply helm-charts/FATE/templates/python-spark.yaml]: I changed the file name to be more semantic, `spark-default-template`.

**Review by LaynePeng — APPROVED** [review]
LGTM to me. Cuz:
1. normal volume mount is read-only, so...
2. for consistency, the template file is not used the traditional markup.

**Review by jiahaoc1993 — APPROVED** [review]

**Pull request merged**